server : update auto gen files comments [no ci]

danbev opened the pull request:
This commit updates the 'auto generated files' comments in server.cpp and removes `deps.sh` from the comment.

The motivation for this change is that `deps.sh` was removed in Commit 91c36c269bca75b2d08119c653512cd20b4ea2ba ("server : (web ui) Various improvements, now use vite as bundler (#10599)").

danbev committed:
server : update auto gen files comments [no ci]

This commit updates the 'auto generated files' comments in server.cpp
and removes `deps.sh` from the comment.

The motivation for this change is that `deps.sh` was removed in
Commit 91c36c269bca75b2d08119c653512cd20b4ea2ba ("server : (web ui)
Various improvements, now use vite as bundler (#10599)").

ngxson commented on examples/server/server.cpp:
```diff
@@ -14,7 +14,9 @@
-// auto generated files (update with ./deps.sh)
+// auto generated files update by:
+// $ npm run build --prefix examples/server/webui (updates examples/server/public/index.html.gz)
+// $ cmake --build build --target llama-server    (generates build/examples/server/index.html.gz.hpp)
```
There is a guide already in server/README, so I think we should point to the README instead of copying it here, so in the future we won't forget this again

ngxson replied:
Btw in the README, can you also add the clarification for `cmake` command to generate `index.html.gz.hpp` ?

danbev replied:
I've moved the comments to the README.md in https://github.com/ggerganov/llama.cpp/pull/11484/commits/6d21abfc201ae0f1fad5fe4991785618e17bc3e5.

danbev committed:
squash! server : update auto gen files comments [no ci]

Move comments about file generation to README.md.

danbev committed:
squash! server : update auto gen files comments [no ci]

Remove the comments in server.cpp that mention that information
can be found in the README.md file.
